etc/launch-tae-logservice/dn1.toml

```diff
@@ -0,0 +1,40 @@
+# service node type, [DN|CN|LOG]
+service-type = "DN"
+
+[log]
+level = "info"
+format = "json"
+max-size = 512
+
+
+[hakeeper-client]
+service-addresses = [
+    "127.0.0.1:32001",
+    "127.0.0.1:32011",
+    "127.0.0.1:32021",
+]
+
+[[fileservice]]
+name = "LOCAL"
+backend = "MEM"
+
+[[fileservice]]
+name = "S3"
+backend = "MEM"
+
+[[fileservice]]
+name = "ETL"
+backend = "DISK-ETL"
+data-dir = "store"
+
+[dn]
+uuid = "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf"
+listen-address = "127.0.0.1:41010"
+service-address = "127.0.0.1:41010"
+
+[dn.Txn.Storage]
+# txn storage backend implementation. [TAE|MEM]
+backend = "MEM"
```

Comment:
"MEM" DN is compatible with "memory" CN or "disttae" CN.
So if you want to run a "tae" CN, this file is not necessary.